e2e/helper_test.go

```diff
@@ -76,6 +77,9 @@ func runWithDindSwarmAndRegistry(t *testing.T, todo func(dindSwarmAndRegistryInf
 	cmd.Command = dockerCli.Command("load", "-i", tmpDir.Join("cnab-app-base.tar.gz"))
 	icmd.RunCmd(cmd).Assert(t, icmd.Success)
 
+	cmd.Command = dockerCli.Command("pull", "busybox:1.30.1")
```

Comment:
Why do we need to pre-pull this image? 🤔

Reply:
It's used in a few test cases as part of their compose file. Docker app build won't pull them.